### ISSUE TITLE
 Windows virtual machines cannot be used under nested virtual machines.

### ISSUE THREAD

**Issue by ZhangSetSail** [kubevirt/kubevirt#10678]
**What happened**:
Windows virtual machines cannot be used under nested virtual machines.

**What you expected to happen**:
expected:
<img width="1030" alt="image" src="https://github.com/kubevirt/kubevirt/assets/101104760/4a951fa5-7916-46b1-bc7f-61342de773bd">
but:
<img width="1030" alt="image" src="https://github.com/kubevirt/kubevirt/assets/101104760/af0cb3c1-f6f4-414b-addf-8968192d5f8c">

**How to reproduce it (as minimally and precisely as possible)**:
1. Aliyun purchased an ecs server, which itself is a virtual machine.
2. Upgrade kernel to 5.4
3. Install the latest version of docker
4. deploy k8s
5. install kubevirt in kubevirt namespace
6. execute nested virtualization commands
```
kubectl -n kubevirt patch kubevirt kubevirt --type=merge --patch '{"spec":{"configuration":{"developerConfiguration":{"useEmulation":true}}}}'
```
7.Deploy vm resources
```
apiVersion: kubevirt.io/v1
kind: VirtualMachine
metadata:
  annotations:
    kubevirt.io/latest-observed-api-version: v1
    kubevirt.io/storage-observed-api-version: v1
  creationTimestamp: "2023-11-03T05:32:23Z"
  finalizers:
  - kubevirt.io/virtualMachineControllerFinalize
  generateName: gr7e520e
  generation: 1
  labels:
    kubevirt.io/domain: vm-demo
  name: vm-demo
  namespace: zqh
spec:
  runStrategy: Always
  template:
    metadata:
      creationTimestamp: null
      labels:
        kubevirt.io/domain: image-demo-poi
      name: vm-demo
    spec:
      affinity:
        nodeAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            nodeSelectorTerms:
            - matchExpressions:
              - key: kubernetes.io/arch
                operator: In
                values:
                - amd64
      domain:
        cpu:
          cores: 2
        devices:
          disks:
          - bootOrder: 1
            cdrom:
              bus: sata
            name: vmimage
          - disk:
              bus: sata
            name: manual21
          interfaces:
          - masquerade: {}
            model: e1000
            name: default
        machine:
          type: q35
        resources:
          limits:
            cpu: "4"
            memory: 8Gi
          requests:
            cpu: "4"
      networks:
      - name: default
        pod: {}
      terminationGracePeriodSeconds: 10
      volumes:
      - containerDisk:
          image: goodrain.me/abc:001
        name: vmimage
      - name: manual21
        persistentVolumeClaim:
          claimName: manual21
```
goodrain.me/abc:001 is my windows 10 image
8. vnc

**Additional context**:
Virtual machine shuts down on its own after being started for a while
<img width="1918" alt="image" src="https://github.com/kubevirt/kubevirt/assets/101104760/13898864-708f-405d-aeff-aca9a5ebc2cf">

The same deployment method and vm resources will not cause problems in bare metal servers

**Environment**:
- KubeVirt version (use `virtctl version`): N/A
- Kubernetes version (use `kubectl version`): N/A
- VM or VMI specifications: N/A
- Cloud provider or hardware configuration: N/A
- OS (e.g. from /etc/os-release): N/A
- Kernel (e.g. `uname -a`): N/A
- Install tools: N/A
- Others: N/A

**Comment by mjudeikis** [comment]
We discussed this yesterday on the community call, but while driving home flashback hit me:
```
execute nested virtualization commands
kubectl -n kubevirt patch kubevirt kubevirt --type=merge --patch '{"spec":{"configuration":{"developerConfiguration":{"useEmulation":true}}}}'
```
I don't this this is "nested virtualization". This is emulation, which is different. Nested virtualization should be enabled at Host level to "passthrough" capabilities from underlaying host to VM, so you can run Second layer L2 virtualisation. 

Tested this on Azure:
```
---
apiVersion: kubevirt.io/v1
kind: KubeVirt
metadata:
  name: kubevirt
  namespace: kubevirt
spec:
  workloads:
    nodePlacement:
      nodeSelector:
        agentpool: virt
  certificateRotateStrategy: {}
  configuration:
    developerConfiguration:
      featureGates: 
      - LiveMigration
      - DataVolumes
      - Passt
      - ExperimentalVirtiofsSupport
  customizeComponents: {}
  imagePullPolicy: IfNotPresent
  workloadUpdateStrategy: {}
```

With vm with live migration: https://gist.github.com/mjudeikis/7c3da12d587cd31bf25a1de9d96c7c6b
I was able not only to run it but migrate around the place. I used `https://learn.microsoft.com/en-us/azure/virtual-machines/fsv2-series`  32Core machines which support NestedVirtualization. 

TL:DR: Emulation != NetedVirtualization 

https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/nested-virtualization
https://cloud.google.com/compute/docs/instances/nested-virtualization/overview

**Comment by ZhangSetSail** [comment]
Thanks for the answer
